app/templates/goto.hbs

```diff
@@ -0,0 +1,10 @@
+{{page-title 'Please wait'}}
+
+<section class='redirection-wrapper'>
+  <img
+    class='logo-img'
+    src='/assets/images/rds-logo-2x.png'
+    alt='Real Dev Squad'
+  />
+  <h1 class='redirection'>Redirecting to your correct location...</h1>
+</section>
```

Comment:
Write test

Reply:
Will add msw for testing this in other PR